content/articles/Integrating-mongodb-with -java/index.md

```diff
@@ -0,0 +1,241 @@
+In this article, you'll learn how to connect MongoDB with Java using Eclipse and how to perform CRUD (create, read, update, and delete) operations using Java and the MongoDB Java Driver. MongoDB is based on a document model written in C++, JavaScript, and Python.
+
+### Table of Contents 
+- [Prerequisites](#prerequisite) 
+- [Relational Database Management Systems vs MongoDB](#Relational-Database-Management-Systems-vs-MongoDB)
+-  [Connecting MongoDB with Java](#Connecting-MongoDB-with-Java) 
+- [Creating MongoDB collection](#Creating-MongoDB-collection)
+- [Insert data into the collection](#Insert-data-into-the-collection) 
+- [Update data in the collection](#Update-data-in-the-collection)
+- [Delete data in the collection](#Delete-data-in-the-collection) 
+- [Delete the collection](#Delete-the-collection)
+- [Eclipse versus Intellij Idea in MongoDB integration with Java](#Eclipse-versus-Intellij-Idea-in-MongoDB-integration-with-java)  
+- [Conclusion](#Conclusion)
+ 
+### Prerequisite
+To follow along with this tutorial, you'll need the following: 
+- A basic understanding of the Java language
+- Eclipse IDE installed on your OS- 
+- MongoDB installed on your OS
+- MongoDB Java Connector
+- Java JDK 
+
+### Relational Database Management Systems vs MongoDB
+#### MongoDB is a non-relational database, while Relational Database Management System is a relational database.
+
+Relational databases store data in rows and columns, also known as records. A relational database works by the use of keys to connect information from multiple tables, e.g., foreign keys and primary keys. On the other hand, data get stored in a non-relational database. There are no tables, rows, primary keys, or foreign keys, as there are in a relational database. MongoDB makes use of documents, which get stored in the form of BSON documents.
+MySQL, Oracle, IBM DB2, and Microsoft SQL Server are some of the most popular RDBMS.
+MongoDB, CouchDB, CouchBase, Cassandra, HBase, Redis, Riak, and Neo4J are the popular NoSQL databases.
+
+#### MongoDB uses collections, whereas RDBMS uses tables.
+Collections are how MongoDB organizes documents. In relational databases, collections it makes use of tables.
+[Differences between collections and tables.](https://docs.mongodb.com/manual/core/databases-and-collections/)
+
+
+#### No schema definition is required in MongoDB, but a clear schema is required in RDBMS.
+MongoDB allows unstructured data to be stored that is building your application without first defining the schema. Before adding data to a relational database, you must first define your schema and this affects cchanging the schema structure of a relational database as it'll be expensive and time-consuming. A schema is the structure of the database that is described in a formal language supported by the database management system. Since MongoDB does not require a predefined schema, it is much easier to update as data and requirements are changing.

```

Comment:
This paragraph needs rephrasing.

You are beating around the bush. Please statement your point directly, and explain in simple english.